pkg/controller/datavolume/controller-base.go

```diff
@@ -1164,8 +1164,6 @@ func (r *ReconcilerBase) handlePvcCreation(log logr.Logger, syncState *dvSyncSta
 // shouldUseCDIPopulator returns if the population of the PVC should be done using
 // CDI populators.
 // Currently it will use populators only if:
```

Comment:
Maybe rephrase this too since now there's only prerequisite to use populators. I think it's more accurate to say something like `It will use populators if a storage class with CSI Drivers exist`.

Reply:
I added a second condition about the usePopulator ann, hopefully that is ok